src/Mpdf.php

```diff
@@ -12970,12 +12977,18 @@ function SetFooter($Farray = [], $side = '')
 
 	/* -- WATERMARK -- */
 
-	function SetWatermarkText($txt = '', $alpha = -1)
+	function SetWatermarkText($txt = '', $alpha = -1, $size = 120, $pos = null)
 	{
 		if ($alpha >= 0) {
 			$this->watermarkTextAlpha = $alpha;
 		}
 		$this->watermarkText = $txt;
+		$this->watermark_font_size = $size;
```

Comment:
I really don't like overwriting the config setting. I understand that in the current state of code where everything is used as a global there is not much space to play with, but maybe a different solution could be devised?